lib/lib.es6.d.ts

```diff
@@ -8530,6 +8530,7 @@ interface DOMException {
     readonly URL_MISMATCH_ERR: number;
     readonly VALIDATION_ERR: number;
     readonly WRONG_DOCUMENT_ERR: number;
+    new(message: string, type: DOMExceptionType): DOMException;
```

Comment:
Should both parameter be optional? https://developer.mozilla.org/en-US/docs/Web/API/DOMException/DOMException